src/modules/pivotTable/useParentSize.js

```diff
@@ -20,13 +21,18 @@ export const useParentSize = (
                 height: el.clientHeight,
             })
         }
+
         onResize(el)
 
+        if (renderCounter) {
+            setSize({ width: undefined })
```

Comment:
Hmm... actually, since this replaces the entire `state` object why are we overriding only `width` explicitly?  (`height` is `undefined` by default).  A bit more explicit might be `setSize({})` or `setSize({ width: undefined, height: undefined })` or `setSize({ width: 0, height: 0 })` (I missed before that we're checking for `!width || !height` below, so `0` has the same effect as `undefined` and it's nice to match the `initialSize`. default, probably?)

Reply:
Yes, I'll go with the `initialSize`.